cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -493,6 +493,14 @@ endif()
 
 #---Check for GSL library---------------------------------------------------------------
 if(mathmore OR builtin_gsl)
+  if(builtin_gsl AND NO_CONNECTION)
+    if(fail-on-missing)
+      message(FATAL_ERROR "No internet connection. Please check your connection, or either disable the 'builtin_gsl' option or the 'fail-on-missing' to automatically disable options requiring internet access")
+    else()
+      message(STATUS "No internet connection, disabling 'builtin_gsl' option")
+      set(builtin_gsl OFF CACHE BOOL "Disabled because there no internet connection" FORCE)
```

Comment:
I think "is" is missing; one should read "Disabled because there is no internet connection"
The same for all the other messages.

Reply:
Indeed, thanks @pamputt !